server/routes/lineitem.js

```diff
@@ -0,0 +1,41 @@
+const express = require('express');
+const db = require('../../db/models');
+const LineItem = db.LineItem;
+
+const router = express.Router();
+
+router.get('/', (req, res, next) => {
+    LineItem.findAll()
+    .then((lineitems) => res.json(lineitems))
+    .catch(next);
+});
+
+router.get('/:id', (req, res, next) => {
+    LineItem.findOne({where: {id: req.params.id}})
+        .then((lineitem) => res.json(lineitem))
+        .catch(next);
+    
+});
+
+router.get('/:cartid', (req, res, next) => {
+    LineItem.findAll({where: {cartId: req.params.id}})
```

Comment:
req.params.id should be req.params.cartid, as you have the route written. We should probably also camel case that as cartId.

Reply:
Ah thanks for catching that!